collection/vivek.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Vivek",
+  "meaning": "Discretion",
+  "translations": {
+    "afr": "diskresie",
```

Comment:
Can you please make sure that the translations are actually the name's translations in different languages and not its meaning?